.github/workflows/integration_test.yaml

```diff
@@ -47,3 +47,6 @@ jobs:
       # need to run in sudo mode due to chroot
       - name: Run integration tests
         run: sudo $(which tox) -e integration -- -m amd64 --image=${{ matrix.image }} ${{ secrets.INTEGRATION_TEST_ARGS }}
+      - name: Setup tmate session
+        if: ${{ failure() }}
+        uses: canonical/action-tmate@main
```

Comment:
Looks like something isn't working still?

Reply:
Yup :( the chroot within environment fails to recognize OS certs